src/DetailsView/components/load-assessment-helper.ts

```diff
@@ -33,7 +34,7 @@ export class LoadAssessmentHelper {
                 parsedAssessmentData = this.assessmentDataParser.parseAssessmentData(content);
             } catch {
                 console.log('Invalid JSON');
```

Comment:
We should probably remove the console logs on this and the one below

Reply:
removed